src/components/Table/GridTable.test.tsx

```diff
@@ -1663,8 +1663,8 @@ describe("GridTable", () => {
     click(cell(r, 3, 1).children[0] as any);
 
     // Then the Header and Parent rows should have the `indeterminate` checked value
-    expect(cellAnd(r, 0, 1, "select")).toBePartiallyChecked();
-    expect(cellAnd(r, 1, 1, "select")).toBePartiallyChecked();
+    expect(cellAnd(r, 0, 1, "select")).toHaveAttribute("data-indeterminate", "true");
+    expect(cellAnd(r, 1, 1, "select")).toHaveAttribute("data-indeterminate", "true");
```

Comment:
React-Aria got rid of setting `aria-checked` which was what `toBePartiallyChecked` looks for. 
See comment: https://github.com/adobe/react-spectrum/issues/4004#issuecomment-1414537911